internal/services/compute/proximity_placement_group_resource.go

```diff
@@ -111,6 +153,22 @@ func resourceProximityPlacementGroupRead(d *pluginsdk.ResourceData, meta interfa
 		d.Set("resource_group_name", id.ResourceGroupName)
 
 		d.Set("location", location.Normalize(model.Location))
+
+		if props := model.Properties; props != nil {
+			if intent := props.Intent; intent != nil {
+				if intent.VMSizes != nil {
+					d.Set("intent_vm_sizes", set.FromStringSlice(*intent.VMSizes))
+				}
+			}
+		}
+
+		zone := ""
+		if model.Zones != nil && len(*model.Zones) > 0 {
+			z := *model.Zones
+			zone = z[0]
+		}
+		d.Set("zone", zone)
```

Comment:
we have common schema logic for zones: https://github.com/hashicorp/go-azure-helpers/tree/main/resourcemanager/zones

Reply:
updated